src/modal/index.tsx

```diff
@@ -53,23 +53,33 @@ class Modal extends Component<ModalProps> {
   componentDidMount() {
     if (this.props.isOpen) {
       this.addListeners()
+      this.setDocumentScroll('hidden')
     }
   }
 
   componentDidUpdate(prevProps: ModalProps) {
     if (!prevProps.isOpen && this.props.isOpen) {
       this.addListeners()
+      this.setDocumentScroll('hidden')
     }
 
     if (!this.props.isOpen && prevProps.isOpen) {
       this.removeListeners()
+      this.setDocumentScroll('')
```

Comment:
Shouldn't you set the overflow to "visible"?

Reply:
This way it remove the property